README.md

```diff
@@ -144,6 +144,7 @@
 - [Keka](http://www.kekaosx.com/en/) - Compress to and extract from many archive file formats. [![Open-Source Software][OSS Icon]](http://sourceforge.net/projects/keka/files/)
 - [MacDown](http://macdown.uranusjr.com/) - Markdown editor. [![Open-Source Software][OSS Icon]](https://github.com/uranusjr/macdown)
 - [Mackup](https://github.com/lra/mackup) - Keep your application settings in sync. [![Open-Source Software][OSS Icon]](https://github.com/lra/mackup)
+- [Media Converter](http://media-converter.sourceforge.net/) - Simple (drag and drop) but advanced media conversion. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/media-converter/code/HEAD/tree/)
```

Comment:
There are commercial ones that are not intuitive to use and only support professional video codecs like http://www.sorensonmedia.com/squeeze/squeeze-desktop/. I faintly remember http://www.ffmpegx.com/ and http://thelittleappfactory.com/evom/ but the first one is ancient and the other one does seem to have received much love as of late. My favorite app for video encoding is still HandBrake, see pull request #95.